src/rest.js

```diff
@@ -37,9 +37,14 @@ function find(config, auth, className, restWhere, restOptions, clientSDK) {
 
 // get is just like find but only queries an objectId.
 const get = (config, auth, className, objectId, restOptions, clientSDK) => {
+  var restWhere = { objectId };
   enforceRoleSecurity('get', className, auth);
-  const query = new RestQuery(config, auth, className, { objectId }, restOptions, clientSDK);
-  return query.execute();
+  return triggers.maybeRunQueryTrigger(triggers.Types.beforeFind, className, restWhere, restOptions, config, auth).then((result) => {
```

Comment:
This si a Parse.Query so that makes sense it's a find, I was thinking more of the fetches, or all the calls that start with a get (where we can know it came from the 'get' endpoints, and not on the 'find' endpoints)
Also, we need to be sure as we have CLP that are based on GET vs FIND :)